core/gui/src/app/dashboard/component/admin/execution/admin-execution.component.ts

```diff
@@ -1,65 +1,30 @@
-import { Component, OnDestroy, OnInit } from "@angular/core";
+import { Component, OnDestroy, OnInit, AfterViewInit } from "@angular/core";
 import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
 import { AdminExecutionService } from "../../../service/admin/execution/admin-execution.service";
 import { Execution } from "../../../../common/type/execution";
-import { NzTableFilterFn, NzTableSortFn } from "ng-zorro-antd/table";
+import { NzTableFilterFn } from "ng-zorro-antd/table";
 import { NzModalService } from "ng-zorro-antd/modal";
 import { WorkflowExecutionHistoryComponent } from "../../user/user-workflow/ngbd-modal-workflow-executions/workflow-execution-history.component";
-import { Workflow } from "../../../../common/type/workflow";
 import { WorkflowWebsocketService } from "../../../../workspace/service/workflow-websocket/workflow-websocket.service";
-import { environment } from "../../../../../environments/environment";
 
 @UntilDestroy()
 @Component({
   templateUrl: "./admin-execution.component.html",
+  styleUrls: ["./admin-execution.component.scss"],
 })
-export class AdminExecutionComponent implements OnInit, OnDestroy {
-  Executions: ReadonlyArray<Execution> = [];
-  workflowsCount: number = 0;
-  listOfExecutions = [...this.Executions];
-  workflows: Array<Workflow> = [];
-  executionMap: Map<number, Execution> = new Map();
+export class AdminExecutionComponent implements OnInit, OnDestroy, AfterViewInit {
+  listOfExecutions: ReadonlyArray<Execution> = [];
+  isLoading: boolean = true;
+  totalWorkflows: number = 0;
+  pageSize: number = 6;
```

Comment:
I think it is natural to use a pageSize of 10. Also, add a scrollbar if the window is too small to show all of the 10 results.

Reply:
addressed